clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -253,6 +253,10 @@ Changes in existing checks
   mode by resolving symbolic links to header files. Fixed handling of Hungarian
   Prefix when configured to `LowerCase`.
 
+- Improved :doc:`readability-duplicate-include
```

Comment:
Please keep alphabetical order (by check name) in this section,

Reply:
Ah, sure, I've noticed that `readability-*` checks here are not sorted by name, so just appended at the end 

(Assuming that correct order should be
```
- Improved :doc:`readability-duplicate-include ...
- Improved :doc:`readability-identifier-naming ...
- Improved :doc:`readability-implicit-bool-conversion ...
- Improved :doc:`readability-redundant-inline-specifier ...
```
)

Reply:
Yes, this way.